sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -85,7 +85,7 @@ object DateTimeUtils {
     // SPARK-6785: use Math.floor so negative number of days (dates before 1970)
```

Comment:
nit: Do we need to update this comment since this PR will replace `floor` with `floorDiv`?

Reply:
Updated